common/download.go

```diff
@@ -117,9 +117,18 @@ func (d *DownloadClient) Get() (string, error) {
 	var finalPath string
 	sourcePath := ""
 	if url.Scheme == "file" && !d.config.CopyFile {
+		// This is special case for relative path in this case user specify
+		// file:../ and after parse destination goes to Opaque
+		switch {
+		case url.Path == "" && url.Opaque != "":
+			finalPath = url.Opaque
+		case url.Opaque == "" && url.Path != "":
+			finalPath = url.Path
+		case url.Opaque != "" && url.Path != "":
```

Comment:
`url.Opaque` is string so it seems like the second two cases are the same. I think this can be written more simply as:

```
if url.Path != "" {
    // If url.Path is set just use this
    finalPath = url.Path
} else if url.Opaque != "" {
    // otherwise try url.Opaque
    finalPath = url.Opaque
}
```